include/veriblock/pop/blockchain/pop/vbk_block_tree.hpp

```diff
@@ -25,6 +26,18 @@ struct VTBInvalidationInfo {
   BtcBlock::hash_t missing_btc_block;
 };
 
+template <>
+inline void BaseBlockTree<BtcBlock>::decreaseAppliedBlockCount(size_t) {
+  // do nothing
+  // BTC tree is not protected
+}
+
+template <>
+inline void BaseBlockTree<BtcBlock>::increaseAppliedBlockCount(size_t) {
+  // do nothing
+  // BTC tree is not protected
+}
```

Comment:
I have moved it, because in the vbk_block_tree we have a definition of the VbkBlock and BtcBlock with which we specialized function and in the base_block_tree we do not have such definition